src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/DWMColorization.cs

```diff
@@ -0,0 +1,682 @@
+using System;
+using System.Diagnostics;
+using System.Windows.Appearance;
+using System.Windows.Media;
+using Microsoft.Win32;
+using MS.Internal;
+using System.Runtime.InteropServices;
+
+namespace System.Windows;
+internal static class DWMColorization
+{
+    /// <summary>
+    /// Maximum <see cref="Byte"/> size with the current <see cref="Single"/> precision.
+    /// </summary>
+    private static readonly float _byteMax = (float)Byte.MaxValue;
```

Comment:
Using language keywords instead of BCL types

```suggestion
    private static readonly float _byteMax = (float)byte.MaxValue;
```

See https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

> 11. We use language keywords instead of BCL types

Reply:
Indeed forgot to mention that, there was quite a few instances of `Int32` and such.